types/errors/errors.go

```diff
@@ -220,14 +220,14 @@ func isNilErr(err error) bool {
 	return reflect.ValueOf(err).IsNil()
 }
 
-// Wrap extends given error with an additional information.
+// Extend an error with additional information.
 //
-// If the wrapped error does not provide ABCICode method (ie. stdlib errors),
+// If the original error does not provide ABCICode method (ie. stdlib errors),
 // it will be labeled as internal error.
 //
 // If err is nil, this returns nil, avoiding the need for an if statement when
 // wrapping a error returned at the end of a function
-func Wrap(err error, description string) error {
+func Extend(err error, description string) error {
```

Comment:
Maybe an explanation or an example of what you mentioned how it looks when string can be nice.

Reply:
Good point! I've added examples for both `Extend{,f}`.